app/controllers/admin_controller.rb

```diff
@@ -102,4 +104,29 @@ def culled_games
   def about
   end
 
+  def stats_display
+    games = Event.current.top_games()
+    titles = Game.where(culled: false).size
+    active_checkouts = Checkout.where(event: Event.current, closed: false).size
```

Comment:
Nit: create a scope for the current event instead:

In Checkout model:
```ruby
class Checkout < ApplicationRecord
  scope :for_current_event, -> { where(event: Event.current) }
end
```

Here:
```ruby
Checkout.for_current_event.count
```